### ISSUE TITLE
Add the FoxTunes version somewhere

### ISSUE THREAD

**Issue by DJCALIEND** [aidan-g/FoxTunes#77]
At launch at the top for example or at the right bottom with BASS version.

**Comment by aidan-g** [comment]
The version is available at `Publication.Version`.
The BASS version is provided by `PlaybackDetails.xaml` so let's edit this control to also render the program name/version.

Use a `StackPanel` instead of a Grid and add a second `TextBlock`;
For example (I'm guessing...);

```xml
<TextBlock>
    <Run Text="{x:Static Core:Publication.Product}"></Run>
    <Run Text="&nbsp;"></Run>
    <Run Text="{x:Static Core:Publication.Version}"></Run>
</TextBlock>
```

It might be possible to not use a StackPanel and instead replace the existing Text binding with multiple Run elements. The current Grid could then also be removed. Less nesting in the visual tree is good.

**Comment by aidan-g** [comment]
This might be a good time to fix the TODO in `utils.js`;

```js
function version() {
    //TODO: Make this return the actual version somehow.
    return "Fox Tunes";
}
```

This would involve making `utils.js` a text template, see one of the .tt files to see how they work.
For example (I'm guessing..);

```js
function version() {
    return "<%= Publication.Product + " " + Publication.Version %>
}
```

**Comment by aidan-g** [comment]
Fixed as of 6363427d8b2ef7e5a70e348340a58297f2f0ad8a
